root/etc/e-smith/events/actions/nethserver-antivirus-init-avdb

```diff
@@ -22,6 +22,9 @@
 db_dir=/var/lib/clamav
 eicar_db=${db_dir}/eicar.ndb
 
+# set safe permission for config file because of proxy password
+chmod 600 /etc/freshclam.conf
```

Comment:
You can't change permissions on a file from the RPM because: it will be overwritten on the next update.

We can't prevent password leak in this case.